$MirOS: ports/mailnews/pine/patches/patch-imap_Makefile,v 1.5 2005/12/16 14:54:33 tg Exp $
$OpenBSD: patch-Makefile,v 1.5 2003/10/05 22:00:00 jakob Exp $

Second chunk must contain @ETC@ twice

--- imap/Makefile.orig	Sat Apr 30 20:49:58 2005
+++ imap/Makefile	Thu Dec 15 13:51:39 2005
@@ -180,7 +180,7 @@ SSLTYPE=nopwd
 # 4	(default) IPv4 support only
 # 6	IPv6 and IPv4 support
 
-IP=4
+IP=6
 
 
 # The following extra compilation flags are defined.  None of these flags are
@@ -311,7 +311,7 @@ aos art asv aux bsd cvx dpx dyn isc pyr 
 
 bsf bso:	an
 	$(BUILD) BUILDTYPE=$@ \
-	SPECIALS="GSSDIR=/usr SSLDIR=/usr SSLINCLUDE=/usr/include/openssl SSLCERTS=/etc/ssl/certs SSLKEYS=/etc/ssl/private LOCKPGM=/usr/sbin/mlock"
+	SPECIALS="GSSDIR=/usr SSLDIR=/usr SSLINCLUDE=/usr/include/openssl SSLCERTS=@ETC@/ssl SSLKEYS=@ETC@/ssl/private LOCKPGM=${PREFIX}/libexec/mlock"
 
 cyg:	an
 	$(BUILD) BUILDTYPE=cyg \
@@ -475,8 +475,6 @@ sslunix sslsco:
 	@echo ++ SSLTYPE=$(SSLTYPE).nopwd
 	@echo +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 	@echo
-	@echo Do you want to continue this build anyway?  Type y or n please:
-	@$(SH) -c 'read x; case "$$x" in y) exit 0;; *) exit 1;; esac'
 
 sslnone:
 	@echo +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
